src/pretalx/agenda/templates/agenda/base.html

```diff
@@ -13,4 +13,19 @@
 {% endblock %}
 
 {% block content %}
+{% if request.event.settings.show_schedule or request.is_orga %}
```

Comment:
It works great for hiding the talk and agenda pages, but the button on the front page leading to them is still there. Clicking it shows a blank page.

Reply:
I'll put the fix for this in the permissions branch, so that it does not get lost: 42d5b9db